guides/plugins/themes/add-additional-infos-about-your-theme.md

```diff
@@ -0,0 +1,20 @@
+# Add additional infos abour your theme
+
+## Overview
+
+The bigger your theme gets, the more complex it can become. Sometimes the help texts of your theme config are no longer sufficient to describe the possibilities in your theme. Sometimes it is necessary to explain the interaction of your variables or to simply provide the user of your theme with examples and tutorials so that he can use the full scope of your theme.
+
+## Add a info html page
+
+This feature is availabe for both apps and plugins.
+
+## Assets and links
+
+For security reasons, such as preventing XSS attacks, your theme info is displayed as a sandboxed iframe in the administration. Unfortunately, this also means that you cannot display embedded content that uses Javascript, such as YouTube videos.
+
+## Add translations to your theme info
+
```

Comment:
It this PR already ready to review or still a draft? I assume there are some things missing here, yet? 🙂

Reply:
Oh wow... this file shouldn't be commited at all. it's from another feature which is on hold until 6.4.0.0 is released.

Thanks for the hint

Reply:
Gladly. 🙂  Apart from that, is this MR ready to review? I'm about to try getting more traction on it. So it doesn't need to lie around that long, anymore.